test/utils/config.js

```diff
@@ -55,10 +52,8 @@ describe('config utils', () => {
 
 			it('should log error when it fails to write', () => {
 				fs.mkdirSync.throws(new Error('failed to create folder'));
-				getConfig(defaultPath);
-				expect(process.exit).to.be.calledWithExactly(1);
-				return expect(logger.error).to.be.calledWithExactly(
-					`ERROR: Could not write to \`${defaultPath}\`. Your configuration will not be persisted.`,
+				return expect(() => getConfig(defaultPath)).to.throw(
```

Comment:
In other test suite, we do something like `getConfig.bind(null, defaultPath)`
so as other places below.